# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1271110078 14400
#      Mon Apr 12 18:07:58 2010 -0400
# Node ID 153e6226a669167daab31d94cbd617d32decfe89
# Parent  62ebba45054e27eb75e967371ccd4db20b65845d
loglog.m: Minor ticks on by default for loglog plots.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,11 @@
-2010-04-12  Ben Abbott <bpabbott@mac.com>
-
+2010-04-12  Ben Abbott <bpabbot@mac.com>
+
+	* plot/loglog.m: Minor ticks on by default for loglog plots.
 	* plot/__plt_get_axis_arg__.m, plot/newplot.m: Respect the nextplot
 	property value of 'new' for axes and 'replacechildren' for axes and
 	figures.
 
 2010-04-10  Ben Abbott <bpabbot@mac.com>
 
 	* plot/__go_draw_figure__.m: Don't set background for non-visible axes.
 
diff --git a/scripts/plot/loglog.m b/scripts/plot/loglog.m
--- a/scripts/plot/loglog.m
+++ b/scripts/plot/loglog.m
@@ -32,19 +32,23 @@ function retval = loglog (varargin)
   [h, varargin] = __plt_get_axis_arg__ ("loglog", varargin{:});
 
   oldh = gca ();
   unwind_protect
     axes (h);
     newplot ();
 
     set (h, "xscale", "log", "yscale", "log");
+    if (any( strcmp (get (gca, "nextplot"), {"new", "replace"})))
+      set (h, "xminortick", "on", "yminortick", "on");
+    endif
 
     tmp = __plt__ ("loglog", h, varargin{:});
 
     if (nargout > 0)
       retval = tmp;
     endif
   unwind_protect_cleanup
     axes (oldh);
   end_unwind_protect
 
 endfunction
+
